install.sh

```diff
@@ -9,6 +13,7 @@ needed_package_name=null_package_name
 stage=release
 interactive=1
 skip_install=0
+skip_checks=0
```

Comment:
might be better to explicitly name this skip_timedrift_check since we have multiple other checks (like running collectd and collectd installed) which aren't covered by this flag.